### DISCUSSION TITLE
Transition that can be triggered by multiple events?

### DISCUSSION THREAD

**Discussion by MartinOrell** [secure-dev-ops/code-realtime#19] [Q&A] [answered]
In the documentation about Class with State Machine it is stated:
"The same transition can be triggered by multiple trigger operations (just like a transition in a capsule state machine can be triggered by multiple events)."

I'm not sure how "a capsule state machine can be triggered by multiple events".
Let's say we have:
```
transitionA: State1->State2 on timer1.timeout;
transitionB: State1->State2 on timer2.timeout;
```
How do we change that to instead be one transition that can either be triggered by timer1 or timer2? I can't just write
```
sharedTransition: State1->State2 on (timer1.timeout or timer2.timeout);
```

**Comment by mattias-mohlin** [comment, marked as answer]
You should separate the triggers with a comma. So you can write
```
sharedTransition: State1->State2 on timer1.timeout, timer2.timeout;
```

**Reply by MartinOrell** [reply]
ok, that works, thanks!
